src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -1459,12 +1461,14 @@ ObjectDataHandler::read_ret ObjectDataHandler::read(
       ceph_assert(!object_data.is_null());
       ceph_assert((obj_offset + len) <= object_data.get_reserved_data_len());
       ceph_assert(len > 0);
-      laddr_t l_start =
+      auto l_start =
```

Comment:
Suggest not to use `auto` if it is not helpful for readabilty, use explicit `laddr_offset_t l_start` instead.

Use `auto` only if the type is too verbose to read.